### ISSUE TITLE
[Bug]: Introspection Response is missing client_id

### ISSUE THREAD

**Issue by maennchen** [zitadel/zitadel#8590]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

ZITADEL Cloud

### Database

None

### Describe the problem caused by this bug

For a long time, the introspection response contained a `client_id` property.

(Spec: https://datatracker.ietf.org/doc/html/rfc7662#section-2.2)

Recently (Can't pinpoint it exactly, CI started failing after I hadn't touched the project for a while), the property is no longer present.

### To reproduce

1. Request access token using JWT Profile
2. Introspect Access Token
3. Check Response

### Expected behavior

`client_id` should still be present.

### Additional Context

* Test: https://github.com/erlef/oidcc_plug/blob/ci/test/oidcc/plug/introspect_token_test.exs#L193-L240
* Output: https://github.com/erlef/oidcc_plug/actions/runs/10798920140/job/29953498821#step:7:79

**Comment by muhlemmer** [comment]
Thanks for the report. Introspection takes the `client_id` from the token storage. I recently found we didn't set the `client_id` to the token for JWT Profile grant. This was fixed as part of https://github.com/zitadel/zitadel/pull/8580

> Correctly set the client_id field with machine's username.

 As we didn't have a bug report at the time, I didn't consider sending a separate fix for that.

It should be available after the next release and deployment to cloud, beginning of next week.
